### ISSUE TITLE
Mode collapse

### ISSUE THREAD

**Issue by muxgt** [moono/stylegan-reproduced#4]
Hi! I'm trying to train your model with tpu. Had to remove MirroredStrategy and adapt code a bit. But it somehow degrades to a mode collapse on 256-512 resolution. Like this:
![models_ffhq_batch32_512x512_generated_images_gen_11200](https://user-images.githubusercontent.com/54779582/66574414-d72c7d80-eb7c-11e9-87dc-c2a89735d034.png)
There are 8 workers on tpu so it should be the same as 8 gpus. I used the same batch sizes and learning rates as in the official code but I see that you have smaller batch sizes with the same learning rates. What is a correct way to match batch sizes and learning rate?
